lib/agent0/agent0/hyperdrive/policies/lpandarb.py

```diff
@@ -0,0 +1,642 @@
+"""Agent policy for LP trading that also arbitrage on the fixed rate."""
+from __future__ import annotations
+
+from dataclasses import dataclass
+from typing import TYPE_CHECKING
+
+from fixedpointmath import FixedPoint
+
+from agent0.hyperdrive.state import HyperdriveActionType, HyperdriveMarketAction
+from elfpy.types import MarketType, Trade
+
+from .hyperdrive_policy import HyperdrivePolicy
+
+if TYPE_CHECKING:
+    from ethpy.hyperdrive import HyperdriveInterface
+    from numpy.random._generator import Generator as NumpyGenerator
+
+    from agent0.hyperdrive.state import HyperdriveWallet
+
+import time
+
+# pylint: disable=too-many-parameters, too-many-local-variables
+
+# constants
+fp0 = FixedPoint(0)
```

Comment:
I really don't like this pattern because 1) it requires that the person reading your code knows your naming scheme or navigates to the top of the code to see what they mean, 2) it clutters up the preamble of the file 3) it makes the equations look more cryptic on first read because of the short variable names.

I know this comes down to a matter of opinion, and there is nothing technically wrong with this. But I ask that you just go with my opinion until you can convince the team to change the practice throughout the repo. More important than anything else, I'd like us to stay as consistent as possible with our programming patterns.